### PR TITLE
scx_layered: Implement layer config xllc_mig_min_us

### PR DESCRIPTION
**Pull request by htejun**
Now that per-DSQ queued runtimes are tracked, we can modulate cross-LLC
migrations based on them. Add a per-layer config option to skip cross-LLC
migrations if the queued runtime is below the configured value. Default to
100us which should be a safe value in most cases.

### CODE REVIEW

**Review by hodgesds — APPROVED** [review]
Might need to appease the linters, otherwise LGTM.

**Review by JakeHillion — APPROVED** [review]
Nice! Looking forward to trying this out. Will close #944 which attempted something similar but is way out of date.

File: scheds/rust/scx_layered/src/stats.rs [hunk 1]
```diff
@@ -301,12 +304,13 @@ impl LayerStats {
 
         writeln!(
             w,
-            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig={} affn_viol={}",
+            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig/skip={}/{} affn_viol={}",
```
Comment (hodgesds) [inline scheds/rust/scx_layered/src/stats.rs:307]: the `skip` might get confusing down the road, but I think it's fine for now as long as numa/llc related states are grouped together.

**Comment by htejun** [comment]
> Nice! Looking forward to trying this out. Will close #944 which attempted something similar but is way out of date.

We'll probably need a better way to configure this than an absolute limit. Hopefully, we'll have a better idea on what that should be once we gain more experience with this.

**Pull request merged**